docs/content/operations/api-reference.md

```diff
@@ -230,6 +230,20 @@ Enables all segments of datasource which are not overshadowed by others.
 
 Enables a segment of a datasource.
 
+##### PUT
+
+* `/druid/coordinator/v1/datasources/{dataSourceName}/markUnused`
+
+Marks segments unused for a datasource by interval or set of segment Ids. The request payload contains the interval or set of segment Ids to be marked unused.
```

Comment:
The docs need to call out what happens when both are supplied, and the inclusive/exclusive behavior of the intervals

Reply:
if both are supplied, API returns an error `BAD_REQUEST`, will update docs. Good point about inclusive/exclusive interval, how about [start, end), where start is inclusive and end is exclusive , is there a different convention with other Druid APIs, I couldn't find anything mentioned. This also needs to go in proposal i think.

Reply:
It looks like it uses Joda contains in all the other interval based API's. It looks like it should be inclusive of the end in this case: https://www.joda.org/joda-time/apidocs/org/joda/time/base/AbstractInterval.html#contains-org.joda.time.ReadableInterval-

Reply:
@dampcake That's a good point, let's do both inclusive then